### ISSUE TITLE
Dask 2.26.0 breaks test_read_vcfzarr

### ISSUE THREAD

**Issue by ravwojdyla** [pystatgen/sgkit#245]
`dask` was released yesterday version: [2.26.0](https://pypi.org/project/dask/) and it breaks our `test_read_vcfzarr`. That same test works fine on `2.25.0`.

_Originally posted by @ravwojdyla in https://github.com/pystatgen/sgkit/pull/228#issuecomment-691538352_

The issue seems to be that:

```
    ds["variant_id_mask"] = (
        [DIM_VARIANT],
        variants_id == ".",
    )
```

returns `(['variants'], False)` instead of array of bool as the 2nd element.

**Comment by ravwojdyla** [comment]
Further:

```
import dask.array as da

a = da.from_array(["a", "b", ".", "e"])
r = a == "."
print(r)

# 2.26.0: False
# 2.25.0: dask.array<eq, shape=(4,), dtype=bool, chunksize=(4,), chunktype=numpy.ndarray>
```

That's rather big change.

**Comment by ravwojdyla** [comment]
Dask issue: https://github.com/dask/dask/issues/6631

**Comment by ravwojdyla** [comment]
was fixed via #246 closing
